src/explore-education-statistics-frontend/src/modules/table-tool/components/Footnote.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { TableData } from '@common/services/tableBuilderService';
+
+// eslint-disable-next-line @typescript-eslint/prefer-interface
```

Comment:
Don't disable these lints, just turn it into an interface. IntelliJ even automatically fix it for you